outputs.tf

```diff
@@ -15,3 +15,13 @@ output "private_subnet_ids" {
   description = "List of private subnet IDs. These are either supplied in the optional `private_subnets` input variable or created in `subnets` submodule."
   sensitive   = false
 }
+
+output "db_outputs" {
```

Comment:
this will break if databse is not created within the module. The database is optional and the tests don't create it, so tests will fail

Reply:
Latest commit has fixed this issue